yarnitor/api.py

```diff
@@ -19,7 +19,11 @@ def get_model():
 
 @api_bp.route('/api/status')
 def status():
-    return jsonify({'status': 'ok'})
+    ym = get_model()
```

Comment:
Basically this is not a pattern that I've encountered before. This is to be taken with number of grains of salt that would correspond to the vast experience that I have from building two super tiny web apps. basically you can ignore me on this one. I'll sync up with you in person